patch/zcs-patch-8.7.11_GA_P1_1855/README

```diff
@@ -0,0 +1,55 @@
+This patch includes following fixes on ZCS 8.7.11 GA
+ZCS-4115 - Bug 107925 - Persistent XSS - snippet [CWE-79]
+ZCS-4116 - Bug 108265 - Persistent XSS - message view as text [CWE-79]
+
+Please re-check your ZCS version. This patch should only be applied on 8.7.11 GA.
+In case of any issues after patch, you can revert the changes by ./install_patch -r
+
+-------------  To Install the Patch -------------
+
+root@zqa-036:/# tar -xvzf zcs-patch-8.7.11_P1_GA_1855.tgz
```

Comment:
what is 1855?   
the naming convention 
zcs-patch-8.7.11_GA_P1_XXXXX

Reply:
8.7.11 GA is having 1854 so incremented it by one